package.json

```diff
@@ -55,31 +58,69 @@
   },
   "devDependencies": {
     "@semantic-release/git": "^7.0.5",
+    "@types/jest": "^23.3.13",
+    "@types/node": "^10.12.18",
+    "@types/ora": "^3.0.0",
     "babel-eslint": "7.1.1",
-    "coveralls": "^3.0.0",
+    "husky": "^1.3.1",
```

Comment:
Is husky still needed? IIRC we removed all the husky scripts.

Reply:
We are using it -- see https://github.com/infinitered/ignite/pull/1379/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R104

```json
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
```